static/js/settings.js

```diff
@@ -0,0 +1,43 @@
+//DOM
+const save_settings = document.getElementById('save_settings')
+const fire = document.getElementById('fire')
+const activate_modals = document.getElementsByClassName('activate_modal')
+
+//GLOBAL
+state = 0
+
+//MODALS - Event Listener 
+for (const button of activate_modals) {
+    button.addEventListener('click', function(e) {
+        const type = e.target.id
+        const modal = document.querySelector(`.${type}`)        
+        modal.style.visibility = 'visible' 
+        const close = modal.firstElementChild.children[0]
+        close.addEventListener('click', function() {
+            modal.style.visibility = 'hidden'
+        })
+    });
+}
+
+//SETTINGS
+save_settings.addEventListener('click', function() {
+    const animate_bg = document.getElementById('animate_bg')
+    const bodyEle = document.querySelector('body')
```

Comment:
nit: Ele isn't useful, I think convention is `body` or `bodyElement` or `bodyEl`.